.gitignore

```diff
@@ -1,5 +1,6 @@
 #$ build dir
 build_
+course5/builds
```

Comment:
This is one of the reasons I believe it's better to keep your builds outside of your source directory